docs/self-managed/setup/deploy/amazon/amazon-eks/eksctl.md

```diff
@@ -291,148 +298,256 @@ For detailed examples, review the [documentation provided by AWS](https://docs.a
 
-12. Wait for changes to be applied:
+   Ensure that the `engine-version` matches the previously created PostgreSQL cluster.
 
-```shell
-aws rds wait db-instance-available \
-    --db-instance-identifier $RDS_NAME
-```
+   ```shell
+   aws rds create-db-instance \
+       --db-instance-identifier $RDS_NAME \
+       --db-cluster-identifier $RDS_NAME \
+       --engine aurora-postgresql \
+       --engine-version $POSTGRESQL_VERSION \
+       --no-publicly-accessible \
+       --db-instance-class db.t3.medium
+   ```
 
-### Verifying connectivity between the Amazon EKS cluster and the PostgreSQL database
+   - More configuration options can be found in the [AWS documentation](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/rds/create-db-instance.html).
 
-1. Retrieve the writer endpoint of the DB cluster.
+9. **Wait for changes to be applied:**
 
-```shell
-export DB_HOST=$(aws rds describe-db-cluster-endpoints \
-  --db-cluster-identifier $RDS_NAME \
-  --query "DBClusterEndpoints[?EndpointType=='WRITER'].Endpoint" \
-  --output text)
-```
+   ```shell
+   aws rds wait db-instance-available \
+       --db-instance-identifier $RDS_NAME
+   ```
 
-2. Start Ubuntu container in interactive mode within the Amazon EKS cluster.
+### Verifying Connectivity Between the Amazon EKS Cluster and the PostgreSQL Database
 
-```shell
-kubectl run ubuntu --rm -i --tty --image ubuntu --env="DB_HOST=$DB_HOST" --env="PG_USERNAME=$PG_USERNAME" -- bash
-```
+1. **Retrieve the writer endpoint of the DB cluster:**
 
-3. Install required dependencies:
+   ```shell
+   export DB_HOST=$(aws rds describe-db-cluster-endpoints \
+     --db-cluster-identifier $RDS_NAME \
+     --query "DBClusterEndpoints[?EndpointType=='WRITER'].Endpoint" \
+     --output text)
+   ```
 
-```shell
-apt update && apt install -y postgresql-client
-```
+2. **Start an Ubuntu container in interactive mode within the Amazon EKS cluster:**
 
-4. Connect to PostgreSQL database:
+   ```shell
+   kubectl run ubuntu --rm -i --tty --image ubuntu --env="DB_HOST=$DB_HOST" --env="PG_USERNAME=$PG_USERNAME" -- bash
+   ```
 
-```shell
-psql \
-  --host=$DB_HOST \
-  --username=$PG_USERNAME \
-  --port=5432 \
-  --dbname=postgres
-```
+3. **Install required dependencies:**
+
+   ```shell
+   apt update && apt install -y postgresql-client
+   ```
+
+4. **Connect to the PostgreSQL database:**
+
+   ```shell
+   psql \
+     --host=$DB_HOST \
+     --username=$PG_USERNAME \
+     --port=5432 \
+     --dbname=postgres
+   ```
+
+   Verify that the connection is successful.
+
+## OpenSearch Domain
+
+Creating an OpenSearch domain can be accomplished in various ways. This guide provides a reproducible setup using the AWS CLI. For additional information on using the AWS Management Console, refer to the [OpenSearch documentation](https://opensearch.org/docs/latest/index/).
+
+The resulting OpenSearch domain is intended for use with the Camunda platform.
```

Comment:
here is the other occurrence that I mentioned above about the `Camunda 8 platform` vs the `Camunda platform`.